db/sql/0_msar.sql

```diff
@@ -1878,8 +1913,8 @@ DECLARE
   constraint_defs __msar.con_def[];
 BEGIN
   fq_table_name := format('%s.%s', __msar.get_schema_name(sch_oid), quote_ident(tab_name));
-  column_defs := msar.process_col_def_jsonb(null, col_defs, false, true);
-  constraint_defs := msar.process_con_def_jsonb(null, con_defs);
+  column_defs := msar.process_col_def_jsonb(0, col_defs, false, true);
+  constraint_defs := msar.process_con_def_jsonb(0, con_defs);
```

Comment:
I really got confused as the function name is very similar to `msar.process_col_def_jsonb`. I would prefer to call it as `msar.process_column_def`. Just wanted to put my thoughts out, no need to take any action now.

Reply:
I recognize that in this case the abbreviations are a little confusing. I'd rather make more distinguishable abbreviations in this case rather than lengthening the function names. As described elsewhere, horizontal space is at a premium for function signatures (including names). I'll try to think of something eaiser.